mobile/lib/src/views/pipeline_detail_page.dart

```diff
@@ -75,9 +68,8 @@ class _PipelineDetailPageState extends State<PipelineDetailPage> {
                         onToggle: (value) {
                           setState(() {
                             pipeline.enabled = !pipeline.enabled;
+                            GetIt.I<AerisAPI>().updatePipeline(pipeline);
```

Comment:
This won't work if the updatePipeline awaits before changing the state. You should not have unawaited calls in a state call.